build.sbt

```diff
@@ -5,16 +5,48 @@ name := "commercetools-sunrise-cms"
 
 organization := "io.commercetools.sunrise"
 
+/**
+ * PROJECT DEFINITIONS
+ */
+
 lazy val `commercetools-sunrise-cms` = (project in file("."))
   .configs(IntegrationTest)
+  .aggregate(`cms-common`, `cms-contentful`)
   .settings(javaUnidocSettings ++ commonSettings ++ commonTestSettings : _*)
 
+lazy val `cms-common` = project
+  .configs(IntegrationTest)
+  .settings(commonSettings ++ commonTestSettings : _*)
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.apache.commons" % "commons-lang3" % "3.0",
```

Comment:
version 3.4 is the most recent

http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.apache.commons%22%20AND%20a%3A%22commons-lang3%22